### DISCUSSION TITLE
Possible transfer of `jonemo/pythonnet-docker` to `pytonnet` org?

### DISCUSSION THREAD

**Discussion by michaelosthege** [pythonnet/pythonnet#1919] [Ideas]
Hi @filmor, @lostmsu and collaborators!

You might have seen https://github.com/jonemo/pythonnet-docker where @jonemo maintained Docker images for various combinations of Python/pythonnet/Mono.
He approached me asking if I would take over ownership of it, and I'm willing to do that.

But before we start transferring the repository, we wanted to approach you (the `pythonnet` devs) if you'd be willing to have https://github.com/jonemo/pythonnet-docker transferred into the https://github.com/pythonnet organization?
I would be doing the maintenance there, for example updating the images once `pythonnet` version 3 gets released.
Having this in an organization instead of a personal account also makes it easier to have multiple people with "admin" access which is needed to configure secrets such as the ones needed to push images from GH actions into the Docker hub.

What do you think?

best
michael

**Comment by lostmsu** [comment]
@filmor seems like a good idea to me.

**Reply by jonemo** [reply]
Hi, just wanted to add a quick message here to confirm that I support this and am ready to transfer the repo as soon as we have a confirmed future home.

Please also consider what the repo should be named after the transfer. The current name `pythonnet-docker` would shadow the archived repo https://github.com/pythonnet/pythonnet-docker.

**Reply by michaelosthege** [reply]
@filmor @lostmsu did you get a chance to discuss/decide about our proposal already?
